vault/token_store.go

```diff
@@ -419,16 +468,31 @@ func (ts *TokenStore) paths() []*framework.Path {
 		{
 			Pattern: "lookup-self$",
 
+			DisplayAttrs: &framework.DisplayAttributes{
+				OperationPrefix: operationPrefixToken,
+				OperationVerb:   "lookup",
```

Comment:
should this be `lookup-self` to differentiate between `lookup`

Reply:
There are two suffixes for individual operations below (self/self2)